### PR TITLE
Add missing letters for Tigrinya, Tigre, and Blin

### PR DESCRIPTION
**Pull request by fgaim**
This PR adds some missing letters for the Tigrinya, Tigre, and Blin languages.

### CODE REVIEW

**Comment by fgaim** [comment]
The Travis CI check fails are not related to this particular PR.
Test is failing for previously merged commits on Dagbani and Persian input methods; someone needs to look into the matter.

File: test/jquery.ime.test.fixtures.js [hunk 1]
```diff
@@ -23,6 +23,9 @@ var palochkaVariants = {
 			{ input: 'gui', output: 'ጒ', description: 'Blin gui -> ጒ' },
 			{ input: 'guie', output: 'ጔ', description: 'Blin guie -> ጔ' },
 			{ input: 'gW', output: 'ጕ', description: 'Blin gW -> ጕ' },
+			{ input: 'Gui', output: 'ⶔ', description: 'Blin Gui -> ⶔ' },
```
Comment (amire80) [inline test/jquery.ime.test.fixtures.js]: This fails. The result is ጙኢ and not ⶔ.

File: test/jquery.ime.test.fixtures.js [hunk 2]
```diff
@@ -23,6 +23,9 @@ var palochkaVariants = {
...
+			{ input: 'Guie', output: 'ⶕ', description: 'Blin Guie -> ⶕ' },
```
Comment (amire80) [inline test/jquery.ime.test.fixtures.js]: This fails. The result is ጙኤ.

File: test/jquery.ime.test.fixtures.js [hunk 3]
```diff
@@ -23,6 +23,9 @@ var palochkaVariants = {
...
+			{ input: 'GW', output: 'ⶖ', description: 'Blin GW -> ⶖ' },
```
Comment (amire80) [inline test/jquery.ime.test.fixtures.js]: This fails. The result is ጝW.

**Comment by fgaim** [comment]
Thanks for reviewing. I just looked into it, the tests actually pass as per the Test Suite, when ran locally. Moreover, I also get the expected letters when using the it myself in the example page. Could you check again? BTW, how are you producing these results? I can't find them in the TravisCI report.

**Comment by amire80** [comment]
> Thanks for reviewing. I just looked into it, the tests actually pass as per the Test Suite, when ran locally. Moreover, I also get the expected letters when using the it myself in the example page. Could you check again?

🤦‍♂️ 

My mistake.

It actually does work. I was running it on a wrong cached version.

I'm going to merge it.

> BTW, how are you producing these results? I can't find them in the TravisCI report.

Yeah, the CI is messed up and we really need to fix it. I usually run the tests in a browser by running a local webservice pointing it to http://localhost/test/ .

**Pull request merged**